### PR TITLE
Minor README.md fixes

### PR DESCRIPTION
**Pull request by isimluk**

### CODE REVIEW

**Comment by redhatrises** [comment]
@isimluk looks like there is an issue with pylint.

**Comment by isimluk** [comment]
I noticed. Thanks God we have him.

File: README.md [hunk 1]
```diff
@@ -11,8 +11,7 @@ Currently available backends are:
...
 | Azure | Pushes events to Azure Log Analytics | <ul><li>[Deployment to AKS](docs/aks)</li></ul> | [Azure backend](fig/backends/azure) |
-| Chronicle | Pushes events to Google Chronicle | <ul><li>[Deployment to GKE](docs/listings/gke-chronicle/UserGuide.md) (using [marketplace](https://console.cloud.google.com/marketplace/product/crowdstrike-saas/falcon-integration-gateway-chronicle))</li><li>[Deployment to GKE](docs/gke-chronicle) (manual)</li></ul> | [Chronicle backend](fig/backends/chronicle) |
-| GCP | Pushes events to GCP Security Command Center | <ul><li>[Deployment to GKE](docs/listings/gke/UserGuide.md) (using [marketplace](https://console.cloud.google.com/marketplace/product/crowdstrike-saas/falcon-integration-gateway-scc))</li><li>[Deployment to GKE](docs/gke) (manual)</li></ul> | [GCP backend](fig/backends/gcp) |
+| Chronicle | Pushes events to Google Chronicle | <ul><li>[Deployment to GKE](docs/listings/gke-chronicle/UserGuide.md) (using [marketplace](https://console.cloud.google.com/marketplace/product/crowdstrike-saas/falcon-integration-gateway-chronicle))</li><li>[Deployment to GKE](docs/chronicle) (manual)</li></ul> | [Chronicle backend](fig/backends/chronicle) |
```
Comment (shawndwells) [inline README.md:14]: Did we drop the GCP Security Command Center integration? Or was this always a duplicate of Chronicle?

Comment (isimluk) [reply README.md:14]: GCP is bellow. GCP was the duplicate line. (but I also fixed broken link in Chronicle.

**Pull request merged**